--- a/Android/android.py
+++ b/Android/android.py
@@ -176,7 +176,8 @@
-    command = [relpath(PYTHON_DIR / "configure")]
+    command = [relpath(PYTHON_DIR / "configure"), "--enable-optimizations"]
     if context.args:
         command.extend(context.args)
     run(command)
@@ -224,8 +225,10 @@
         # Android always uses a shared libpython.
-        "--enable-shared",
-        "--without-static-libpython",
+        "--enable-shared",
+        "--without-static-libpython",
+        "--enable-optimizations",
 
         # Dependent libraries. The others are found using pkg-config: see
         # android-env.sh.
         f"--with-openssl={prefix_dir}",
